ds-ux-guidelines/src/ds-components/navigation/navigation.scss

```diff
@@ -0,0 +1,60 @@
+@mixin menu-list-ul-overide {
```

Comment:
would it make sense to make a comment next to the !important as to the why we are doing it?